### ISSUE TITLE
[iOS] Knob shadow of the slider component gets cut off by left side

### ISSUE THREAD

**Issue by fluiddot** [WordPress/gutenberg#36669]
## Description

The knob component of the slider used in the bottom sheet gets cut off when it's located at the minimum value (left side).

## Step-by-step reproduction instructions

1. Add a Spacer block.
2. Open the block settings.
3. Move the "Height in pixels" slider to the minimum value (left side).
4. Observe that the knob's shadow gets cut off.

## Expected behaviour

The knob's shadow should be rendered properly.

## Actual behaviour

The knob's shadow gets cut off.

## Screenshots or screen recording (optional)

<img src=https://user-images.githubusercontent.com/14905380/142655776-73db6326-2516-4426-9a76-2837928295cf.PNG width=350>

## WordPress information
- WordPress version: N/A
- Gutenberg version: N/A
- Are all plugins except Gutenberg deactivated? N/A
- Are you using a default theme (e.g. Twenty Twenty-One)? N/A

## Device information
- Device: iPhone 11
- Operating system: iOS 15.1
- WordPress app version: `18.7`

**Comment by hypest** [comment]
I have the sense this might have the same root cause as [this other ticket](https://github.com/wordpress-mobile/gutenberg-mobile/issues/4261).

**Comment by fluiddot** [comment]
> I have the sense this might have the same root cause as [this other ticket](https://github.com/wordpress-mobile/gutenberg-mobile/issues/4261).

Yep, most likely. Looks like one of the container views that render the content of the Panel component ([reference](https://github.com/WordPress/gutenberg/tree/trunk/packages/components/src/panel)) might be producing this issue.

**Comment by tyxla** [comment]
This is no longer relevant after the React Native implementation was removed in #78747. Closing.
